hieradata/dev/wso2/wso2is/pattern-4/default.yaml

```diff
@@ -0,0 +1,374 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+classes:
+  - wso2is
+
+wso2::service_name: wso2is
+wso2::hostname: is540.iam.org
+wso2::mgt_hostname: is540.iam.org
+
+wso2::enable_thrift_service: false
+
+wso2::template_list:
+#  - repository/conf/datasources/bps-datasources.xml
+#  - repository/conf/datasources/metrics-datasources.xml
+  - repository/conf/identity/identity.xml
+  - repository/conf/axis2/axis2.xml
+#  - repository/conf/identity/sso-idp-config.xml
+#  - repository/conf/identity/application-authentication.xml
+#  - repository/conf/identity/EndpointConfig.properties
+#  - repository/conf/datasources/am-datasources.xml
+#  - bin/ciphertool.sh
+
+
+wso2::clustering:
+  enabled: true
+  local_member_host: "%{::ipaddress}"
+  local_member_port: 4000
+  domain: is.wso2.domain
+# WKA membership scheme
+  membership_scheme: wka
+  wka:
+    members:
+      -
+        hostname: 172.30.1.59
+        port: 4000
+
+# Provide the server list for setting up sso
+#wso2::sso_service_providers:
+#    wso2esb:
+#      assertion_consumer_service_url: https://esb.dev.wso2.org:9443/acs
+#      default_assertion_consumer_service_url: https://esb.dev.wso2.org:9443/acs
+#    wso2as:
+#      assertion_consumer_service_url: https://as.dev.wso2.org:9443/acs
+#      default_assertion_consumer_service_url: https://as.dev.wso2.org:9443/acs
+
+wso2::ports:
+  proxyPort:
+    http: 80
+    https: 443
+
+# File list to be copied to the product folder, if there are changes found in the files, the system
+# service will be restarted
+wso2::file_list:
+#  - repository/components/lib/mysql-connector-java-5.1.39.jar
+  - repository/components/lib/%{hiera('wso2::datasources::mysql::connector_jar')}
+#  - repository/conf/security/cipher-tool.properties
+#  - repository/conf/security/cipher-text.properties
+#  - repository/conf/security/secret-conf.properties
+
+# Files to be copied to the product folder when restarting the system service
+#wso2::service_refresh_file_list:
+#  - password-tmp
```

Comment:
Looks like the IS Puppet module is tested against latest `wso2base` while the IS Analytics is done on 1.0.1 release. It would be better to bring both to same level.

Reply:
IS Analytics is done on 1.0.0

Reply:
In that case `wso2::service_refresh_file_list` is not used. This was introduced after 1.0.1 release :)